packages/lib/services/plugins/defaultPlugins.ts

```diff
@@ -0,0 +1,11 @@
+import { defaultPluginSetting, PluginSettings } from './PluginService';
+
+const defaultPluginsId = ['io.github.jackgruber.backup', 'plugin.calebjohn.rich-markdown'];
+
+export default function getDefaultPluginSettings(): PluginSettings {
+	const settings: PluginSettings = {};
+	defaultPluginsId.map((pluginId: string) => {
+		settings[pluginId] = defaultPluginSetting();
+	});
+	return settings;
+}
```

Comment:
I feel this function doesn't do anything useful. Also you are duplicating the list of default plugin IDs here and in the place where settings are defined. That list should only appear once in the codebase.